jobserver/serializers.py

```diff
@@ -22,57 +23,79 @@ class Meta:
             "repo",
             "branch",
             "db",
-            "created_by__username",
+            "owner",
             "created_at",
         ]
         read_only_fields = ["created_at"]
 
 
-class JobSerializer(serializers.HyperlinkedModelSerializer):
+class JobSerializer(serializers.Serializer):
+    """
+    Serializer for a Job as job-runner expects it to look
+
+    Now that various fields have moved off Job and Jobs are grouped by
```

Comment:
Perhaps the serializer should be called `JobShimSerializer`?

The comment could be a bit more explicit, e.g. "a shim to hide our current implementation of JobRequests from the job server, until it is implemented there"?